### ISSUE TITLE
IE8 bug on node engine

### ISSUE THREAD

**Issue by Novak99** [sapegin/grunt-webfont#264]
Hi,
I apologize if this has already been resolved, or if I was so blind not to see the solution.
I use webfont with node engine, in all browsers works fine except in internet explorer 8 (didn't look earlier versions).
When I open the file directly, icons are visible but when I go over virtual host, the icons aren't showing, as can be seen from the attached.
Thanks in advance.
![webfont-ie8-bug](https://cloud.githubusercontent.com/assets/8667018/8795670/207bda52-2f8f-11e5-9409-eca711d6602c.jpg)

**Comment by sapegin** [comment]
Did you try it with other webfonts? Seems like an issue with headers or something like that.

**Comment by Novak99** [comment]
Sorry for delay, yes I tried with other fonts (fontawesome, icomoon...) same problem, looks like ie8 is a problem. I managed to fix it with ie7: true, it fixes ie8 too.
I wanted to avoid that because I didn't want to use defined classes (like bootstrap). 
I want to use it  with mixin where I can call it with "@include icon($position, $icon);" and I tried to avoid  doing it by the hand (defining variables), is it possible to replace class dot "." with "%" sass placeholder in css template, so I can call it with @extend.
Thanks again for your help and time.
